### PR TITLE
Document use of `/` in LinearAlgebra

### LINKED ISSUE #900: use of `/` in LinearAlgebra not documented
Left division using `\` has a nice docstring in LinearAlgebra describing what it does for different linear systems. It would be nice to add a similar explanation for `/`, since the current docstring only describes its behavior for scalar arguments.

### LINKED ISSUE #43607: use of `/` in LinearAlgebra not documented
Left division using `\` has a nice docstring in LinearAlgebra describing what it does for different linear systems. It would be nice to add a similar explanation for `/`, since the current docstring only describes its behavior for scalar arguments.

### PR DESCRIPTION
**Pull request by sasi591**
Closes JuliaLang/LinearAlgebra.jl#900

### CODE REVIEW

File: stdlib/LinearAlgebra/src/generic.jl [hunk 1]
```diff
@@ -1112,6 +1112,30 @@ function (\)(A::AbstractMatrix, B::AbstractVecOrMat)
 end
 
 (\)(a::AbstractVector, b::AbstractArray) = pinv(a) * b
+
+"""
+    /(B, X)
+
+For input matrices/vectors `B` and `X` where the number of columns of `B` and `X` should be same, 
+the resultant matrice `A` such that `A*X ≈ B`. The size of the resultant matrice depends on the row size
+of `B` and row size of `X`.
```
Comment (simonbyrne) [inline stdlib/LinearAlgebra/src/generic.jl:1121]: Typically `X` should refer to the unknown you're solving for. Also, it should refer to `\`:

```suggestion
    A / B

Matrix right-division: `A / B` is equivalent to `(A' \ B')'` where [`\`](@ref) is the left-division operator.
For square matrices, the result `X` is such that `A == X*B`.

See also: [`rdiv!`](@ref).
```

**Comment by DilumAluthge** [comment]
Looks like an error during bootstrap.

**Comment by DilumAluthge** [comment]
Also you need to fix the whitespace errors.

**Comment by sasi591** [comment]
Sorry for the delay, I have made all the changes, thank you. However, I don't understand why the builds are failing.

File: stdlib/LinearAlgebra/src/generic.jl [hunk 2]
```diff
@@ -1112,6 +1112,30 @@ function (\)(A::AbstractMatrix, B::AbstractVecOrMat)
 end
 
 (\)(a::AbstractVector, b::AbstractArray) = pinv(a) * b
+"""
+    A / B
+
+Matrix right-division: `A / B` is equivalent to `(B' \ A')'` where [`\`](@ref) is the left-division operator.
```
Comment (simonbyrne) [inline stdlib/LinearAlgebra/src/generic.jl:1118]: `\`'s need to be escaped:
```suggestion
Matrix right-division: `A / B` is equivalent to `(B' \\ A')'` where [`\\`](@ref) is the left-division operator.
```

**Comment by simonbyrne** [comment]
Thanks!

**Pull request merged**